src/domTree.js

```diff
@@ -25,6 +28,103 @@ const createClass = function(classes: string[]): string {
     return classes.filter(cls => cls).join(" ");
 };
 
+const initNode = function(
+    classes?: string[],
+    options?: Options,
+    style?: CssStyle,
+) {
+    this.classes = classes || [];
+    this.attributes = {};
+    this.height = 0;
+    this.depth = 0;
+    this.maxFontSize = 0;
+    this.style = style || {};
+    if (options) {
+        if (options.style.isTight()) {
+            this.classes.push("mtight");
+        }
+        const color = options.getColor();
+        if (color) {
+            this.style.color = color;
+        }
+    }
+};
+
+/**
+ * Convert into an HTML node
+ */
+const toNode = function(tagName: string): HTMLElement {
+    const node = document.createElement(tagName);
+
+    // Apply the class
+    node.className = createClass(this.classes);
+
+    // Apply inline styles
+    for (const style in this.style) {
+        if (this.style.hasOwnProperty(style)) {
+            // $FlowFixMe Flow doesn't seem to understand span.style's type.
+            node.style[style] = this.style[style];
```

Comment:
Maybe if we type `this.style` as a `CSSStyleDeclaration` it will make `flow` happy.  See https://github.com/facebook/flow/blob/v0.75.0/lib/dom.js#L1418.  That way we only allow valid CSS properties into `this.style`.

Reply:
@kevinbarabash Changing to `CSSStyleDeclaration` seems to give a whole new errors. I'll try to make another PR and if I can't I'll open a new issue.

Reply:
Hmm... so typing a `{}` as a `CSSStyleDeclaration` won't work.  That's too bad.